cmd/info.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2019 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path"
+
+	"github.com/lf-edge/adam/pkg/server"
+	"github.com/spf13/cobra"
+)
+
+var infoCmd = &cobra.Command{
+	Use:   "info",
+	Short: "view info messages",
+	Long:  `View info messages for a specific device, either those already in storage or streaming new`,
+	Run: func(cmd *cobra.Command, args []string) {
+		u, err := resolveURL(serverURL, path.Join("/admin/device", devUUID, "info"))
+		if err != nil {
+			log.Fatalf("error constructing URL: %v", err)
+		}
+		client := getClient()
+		req, err := http.NewRequest("GET", u, nil)
+		if follow {
+			req.Header.Add(server.StreamHeader, server.StreamValue)
+		}
+
+		response, err := client.Do(req)
+		if err != nil {
+			log.Fatalf("error reading URL %s: %v", u, err)
+		}
+		if _, err := io.Copy(os.Stdout, response.Body); err != nil {
```

Comment:
Wait @deitch -- so what's the point of setting -f flag if we're not continuously copying the stream to stdout here?

Reply:
We are. `io.Copy` will keep going until it receives an `io.EOF` from its `Reader`, in this case `response.Body`. As long as it has more data to send - which it will do if the server never sends a "complete" message, it will keep doing it.

Personally, I like the fact that I don't have to worry about it on the client side at all, it just "does the right thing"

Reply:
Awesome! Sorry I missed that subtlety -- let me test it with EVE now.